src/ast_builder.mli

```diff
@@ -1,6 +1,6 @@
 (** Helpers for build OCaml AST fragments *)
 
-open! Import
+open! Import[@@ocaml.warning "-66"]
```

Comment:
Is this necessary? Hasn't `dune` turned that warning off by default now?

Reply:
Indeed (back in May, dune was quite different...).